draft-ietf-tls-tls13.md

```diff
@@ -2247,8 +2277,8 @@ extensions
 
 When this message will be sent:
 
-> A non-anonymous server can optionally request a certificate from the client,
-if appropriate for the selected cipher suite. This message, if sent, will
+> A server which is authenticating with a certificate can optionally
```

Comment:
Actually, didn't we decide in Berlin not to allow PSK + CertificateRequest? (Or maybe that was 0-RTT + CertificateRequest. One of these.) Or am I misremembering? By adding PSK + Sign in this PR, this text now allows it.